Makefile

```diff
@@ -1,5 +1,5 @@
 COMMIT := $(shell git rev-parse --short=7 HEAD 2>/dev/null)
-VERSION := $(shell git describe --abbrev=0 HEAD 2>/dev/null)
+VERSION := $(shell git describe --tags --abbrev=0 HEAD 2>/dev/null)
```

Comment:
Without `--tags` this will only consider annotated tags. I think this was only noticeable when running `make build-image` locally.
```
$  git describe --tags --abbrev=0 HEAD
v0.223.0
$  git describe  --abbrev=0 HEAD
v0.73.0
```